cfgov/v1/tests/jinja2tags/test_jinja2tags.py

```diff
@@ -95,3 +97,58 @@ def test_query_parameter_archived_selected(self):
         s = '{{ is_filter_selected("archived", "exclude") }}'
         template = self.jinja_engine.from_string(s)
         self.assertEqual(template.render({'request': request}), 'True')
+
+
+class TestUniqueIdInContext(TestCase):
+
+    def setUp(self):
+        self.engine = engines['wagtail-env']
+        self.template = '{{ unique_id_in_context() }}'
+        self.random_id_pattern = re.compile('[a-z0-9]{14}')
+
+    def render(self, template, context=None):
+        return self.engine.from_string(template).render(context=context)
+
+    def assert_is_random_id(self, input):
+        match = self.random_id_pattern.match(input)
+        self.assertIsNotNone(match)
+
+    def assert_is_not_random_id(self, input):
+        match = self.random_id_pattern.match(input)
+        self.assertIsNone(match)
+
+    def test_no_context_returns_random_string(self):
+        rendered = self.render(self.template)
+        self.assert_is_random_id(rendered)
+
+    def test_no_request_in_context_returns_random_string(self):
+        rendered = self.render(self.template, {})
+        self.assert_is_random_id(rendered)
+
+    def test_render_with_request_in_context(self):
+        rendered = self.render(self.template, {'request': HttpRequest()})
+        self.assertEqual(rendered, '0')
+        self.assert_is_not_random_id(rendered)
+
+    def test_render_uses_request_to_make_multiple_unique_ids(self):
+        request = HttpRequest()
+        template = ' and '.join([self.template, self.template])
+        self.assertEqual(
+            self.render(template, {'request': request}),
+            '0 and 1'
+        )
+
+    def test_multiple_renders_multiple_unique_ids(self):
+        request = HttpRequest()
+        rendered = [
+            self.render(self.template, {'request': request})
+            for _ in range(5)
+        ]
+        self.assertEqual(rendered, [ '0', '1', '2', '3', '4'])
```

Comment:
```suggestion
        self.assertEqual(rendered, ['0', '1', '2', '3', '4'])
```

I'm surprised that the linter didn't fail on this. Looks like it [ignores the v1 tests directory](https://github.com/cfpb/consumerfinance.gov/blob/e34cc78b3de4627a23c6a981901a2da2da65c785/.flake8#L40). 😳